custombackend/src/androidTest/java/com/algolia/custombackend/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.algolia.custombackend;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
```

Comment:
Let's remove this or do actual tests (I vote for the former 🙃)

Reply:
Removed it for now